src/main/java/edu/greenblitz/tobyDetermined/subsystems/LED.java

```diff
@@ -0,0 +1,65 @@
+package edu.greenblitz.tobyDetermined.subsystems;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class LED extends GBSubsystem {
+
+    private static LED instance;
+    private AddressableLED addressableLED;
+    private AddressableLEDBuffer ledBuffer;
+    private Color defaultColor = Color.kGreen;
```

Comment:
Perhaps define this color in RobotMap..java?

Reply:
I actually think its better inside the subsytem - why do you think it should move?

Reply:
It's not a big deal but I believe default values should be defined in a config file, all in one place so it's easy to find. It's a matter of convention - if someone wants to find a constant, they should probably go look In RobotMap, not the subsystem.

Reply:
The default color is changing in the game so it should be in the LED subsystem. I also think there should be a default default value at robot map

Reply:
By definition, a default doesn't change. If you need it to change throughout the match, change the variable name.